### PR TITLE
Adding visan

### PR DESCRIPTION
**Pull request by svniemeijer**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/main/recipes/example/meta.yaml#L57-L66) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by srepmub** [comment]
count me in as a maintainer ^^

**Comment by svniemeijer** [comment]
I am also (obviously) Ok with being a maintainer.

**Comment by svniemeijer** [comment]
@conda-forge/staged-recipes could you please review this PR?

File: recipes/visan/build.sh [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+
+cmake $CMAKE_ARGS \
+  -DCMAKE_BUILD_TYPE="Release" \
+  -DCMAKE_INSTALL_PREFIX="$PREFIX" \
+  -DCMAKE_PREFIX_PATH="$PREFIX" \
+  -DPython3_EXECUTABLE=$PREFIX/bin/python \
+  ..
+
+make -j$CPU_COUNT
+make install
```
Comment (ocefpaf) [inline recipes/visan/build.sh:16]: Is there a make check/test we can run here?

Comment (svniemeijer) [reply recipes/visan/build.sh:16]: No. This is not available. Visan is mostly a gui application. All the algorithmic elements are in the coda and harp packages that are already on conda-forge.

The tricky part is making sure everything links correctly properly (with the right versions of libraries, etc.), which is why the import tests in the meta file are the most important test for this package.

**Pull request merged**